src/components/ContributionsEpic.tsx

```diff
@@ -3,7 +3,7 @@ import { css } from 'emotion';
 import { body, headline } from '@guardian/src-foundations/typography';
 import { palette } from '@guardian/src-foundations';
 import { space } from '@guardian/src-foundations';
-import { getTrackingUrl } from '../lib/tracking';
+import { addTrackingParams } from '../lib/tracking';
```

Comment:
Just renaming this function for clarity.

Reply:
Ha, I think I find the existing naming clearer. The function doesn't modify an existing value (strings are immutable in JS) it simply builds a tracking URL. `addTrackingParams` doesn't give you clarity over what is returned (if anything), whereas `getTrackingUrl` is very clear - you get a tracking URL back.

Reply:
Yeah it's true we're not strictly "adding" something to string but I'm not sure that particular specific of JS is that relevant here. My thinking as more on these lines - the function takes the base URL as a param, then returns a full URL with tracking params appended to it, so effectively what it does is add the tracking params to the given URL